src/Utilities/FlowAnalysis/Analysis/TaintedDataAnalysis/WebInputSources.cs

```diff
@@ -0,0 +1,732 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.FlowAnalysis.DataFlow;
+using Microsoft.CodeAnalysis.Operations;
+
+namespace Analyzer.Utilities.FlowAnalysis.Analysis.TaintedDataAnalysis
+{
+    internal static class WebInputSources
+    {
+        /// <summary>
+        /// <see cref="SourceInfo"/>s for web input tainted data sources.
+        /// </summary>
+        public static ImmutableList<SourceInfo> SourceInfos { get; }
+
+        /// <summary>
+        /// Statically constructs.
+        /// </summary>
+        static WebInputSources()
```

Comment:
>WebInputSources [](start = 15, length = 15)

Wow, these are lot of hard coded sources :-). Are you planning to add unit tests for all of these or just waiting to move them to an extensible configuration file?

Reply:
My plan is to move to a configuration file, and also test the less straightforward cases (base classes, interface implementations).